tests/02_eradiate/01_unit/scenes/atmosphere/test_heterogeneous.py

```diff
@@ -1,17 +1,24 @@
+from pathlib import Path
+
+import attrs
 import mitsuba as mi
 import numpy as np
 import pytest
+import xarray as xr
 
 import eradiate
 from eradiate import unit_context_config as ucc
 from eradiate import unit_registry as ureg
-from eradiate.contexts import KernelDictContext, SpectralContext
+from eradiate.contexts import CKDSpectralContext, KernelDictContext
+from eradiate.kernel.gridvolume import read_binary_grid3d
 from eradiate.scenes.atmosphere import (
     HeterogeneousAtmosphere,
     MolecularAtmosphere,
     ParticleLayer,
 )
 
+from .test_particle_layer import absorbing_only, scattering_only
```

Comment:
These fixtures are not module-scoped and should rather be defined in a `conftest.py` than in a test module.